pcmanfm/launcher.cpp

```diff
@@ -38,12 +39,26 @@ Launcher::~Launcher() {
 
 }
 
-bool Launcher::openFolder(GAppLaunchContext* /*ctx*/, const Fm::FileInfoList& folderInfos, Fm::GErrorPtr& /*err*/) {
+bool Launcher::openFolder(GAppLaunchContext* ctx, const Fm::FileInfoList& folderInfos, Fm::GErrorPtr& /*err*/) {
     auto fi = folderInfos[0];
     Application* app = static_cast<Application*>(qApp);
     MainWindow* mainWindow = mainWindow_;
     Fm::FilePath path = fi->path();
     if(!mainWindow) {
+        // Launch folders with the default file manager if:
+        //   1. There is no main window (i.e., folders are on desktop),
+        //   2. The folders are not supposed to be opened in new tabs, and
+        //   3. The default file manager exists and is not PCManFM-Qt.
+        if(openWithDefaultFileManager_ && !openInNewTab_) {
```

Comment:
openWithDefaultFileManager is always false.

Reply:
Strange! It works here. But there is a possibility that I should investigate; will tell you soon. Thanks for testing and reviewing!

Reply:
@luis-pereira 
Apart from my installation, that works, I downloaded the branch ["default_file_manager"](https://github.com/lxqt/pcmanfm-qt/tree/default_file_manager), compiled it and tested it without installation on two machines. In all cases, desktop folders were opened by Dolphin when I checked "Open desktop folders in…" and set the default file manager to Dolphin. All changes were applied on-the-fly and correctly.

I have no idea why your test resulted in a different behavior.

Reply:
At the risk of stating the obvious, OK or Apply should be pressed for changes to take effect.